erdpy/delegation/staking_provider.py

```diff
@@ -41,7 +43,9 @@ def prepare_args_for_add_nodes(args: Any):
         validator_pem = validator.get("pemFile")
         validator_pem = path.join(path.dirname(args.validators_file), validator_pem)
         seed, bls_key = parse_validator_pem(validator_pem)
-        signed_message = sign_message_with_bls_key(account.address.pubkey().hex(), seed.hex())
+        if (not args.using_delegation_manager):
```

Comment:
```suggestion
        if not args.using_delegation_manager:
```

Reply:
Parantheses not needed.

Reply:
Yeah, sorry, I'm not a native python speaker.